app/models/manageiq/providers/base_manager/refresher.rb

```diff
@@ -25,6 +25,8 @@ def refresher_options
 
       private
 
+      # FIXME: The intention was to make this method private but tests or code called
+      # from the tests are expecting it to be public.
```

Comment:
Thanks @kbrock, I agree, let's remove this comment now and move this method before private in a followup PR